NerdyNomads/Text-Savvy
# Added link between MongoDB Account objects and Auth0

## What does this PR do?
- An `Account` object will be created in our database whenever a user logs in or signs up through Auth0.
- Changed Account schema (`accounts.model.js`) to contain Auth0 ID, name, and email.
- Removed test forms from `App.js`
 
### Screenshots
This is an example of what is stored in our database:
![image](https://user-images.githubusercontent.com/44079021/156632287-d6d00436-cced-4b32-8608-55f80d7263fb.png)
![image](https://user-images.githubusercontent.com/44079021/156632685-a1f4eccb-2185-412d-aaf1-198281fcb7c4.png)

**Notes:**
- The value of `auth0Id`is taken from Auth0's `user` object.
  - The attribute to access this is `user.sub` and the value takes the form of a string:
    - e.g., "google-oauth2|123abc456789" 

### Steps to test/review
1. Make sure both the frontend and backend are running.
2. Right click -> Inspect.
3. Sign up or log in.
4. In your console, you should get the message "Account added!". You can also check the `Network` tab and see if the `add` request is there.
5. Log out and then log in with the same account. A new account should _not_ have been added.

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
